cpp/src/ray/runtime/abstract_ray_runtime.cc

```diff
@@ -67,8 +67,9 @@ WaitResult AbstractRayRuntime::Wait(const std::vector<ObjectID> &ids, int num_ob
 ObjectID AbstractRayRuntime::Call(RemoteFunctionPtrHolder &fptr,
                                   std::shared_ptr<msgpack::sbuffer> args) {
   InvocationSpec invocationSpec;
-  invocationSpec.task_id =
-      TaskID::ForFakeTask();  // TODO(Guyang Song): make it from different task
+  // TODO(Guyang Song): make it from different task
+  invocationSpec.task_id = TaskID::ForFakeTask();
+  invocationSpec.name = "";
```

Comment:
Not going to expose this until there's something akin to an `AbstractRayRuntime::Options()` API.